chatops/python/docbuilder.py

```diff
@@ -234,7 +234,7 @@ def main():
             except OSError as exception:
                 print_exit('[-] ERR: {0}'.format(exception.strerror),
                            exception.errno)
-        if options['execsummary'] == 'true':  # we're generating a summary as well as a report
+        if options['execsummary'] == True:  # we're generating a summary as well as a report
```

Comment:
(lazy Python programmers leave out the comparison to True :wink: - true pythonista style:
`if options['execsummary`]:` as it's already a boolean test)